fpm.toml

```
@@ -4,6 +4,8 @@ license = "MIT"
 author = "fpm maintainers"
 maintainer = ""
```

```suggestion
maintainer = "@fortran-lang/fpm"
```

Since we're looking at this, it may be a good idea to sync the manifest style to that of `stdlib` (and possibly of other fortran-lang repositories): 

https://github.com/fortran-lang/stdlib/blob/5f347eb763cc99faaac1310bbe4fd742f9db243d/fpm.toml#L1-L6
```suggestion
maintainer = ""
```